packages/gp2-components/src/templates/OutputForm.tsx

```diff
@@ -241,6 +241,7 @@ const OutputForm: React.FC<OutputFormType> = ({
     authors: getPostAuthors(newAuthors),
     tags: newTags.length > 0 ? newTags : undefined,
     mainEntityId,
+    contributingCohorts: newCohorts.length > 0 ? newCohorts : undefined,
```

Comment:
I believe we need to do something like

```
contributingCohorts: newCohorts.length > 0 ? newCohorts.map(cohort => ({ id: cohort.id }) : undefined
```
because the validation assumes that there's only the id in the payload

```
      contributingCohorts: {
        type: 'array',
        items: {
          type: 'object',
          properties: {
            id: { type: 'string' },
          },
          required: ['id'],
        },
        nullable: true,
      },
```

or change the validation otherwise it gives this error

```
{
    "error": "Internal Server Error",
    "message": "{\n  \"status\": 422,\n  \"statusText\": \"Unprocessable Entity\",\n  \"message\": \"Validation error\",\n  \"details\": {\n    \"errors\": [\n      {\n        \"name\": \"unknown\",\n        \"path\": [\n          \"fields\",\n          \"contributingCohorts\",\n          \"en-US\",\n          0,\n          \"id\"\n        ],\n        \"value\": \"5kebYlHRiikmZuANOpMaHm\",\n        \"details\": \"The property \\\"id\\\" is not allowed here.\"\n      },\n      {\n        \"name\": \"unknown\",\n        \"path\": [\n          \"fields\",\n          \"contributingCohorts\",\n          \"en-US\",\n          0,\n          \"name\"\n        ],\n        \"value\": \"AGPDS\",\n        \"details\": \"The property \\\"name\\\" is not allowed here.\"\n      },\n      {\n        \"name\": \"unknown\",\n        \"path\": [\n          \"fields\",\n          \"contributingCohorts\",\n          \"en-US\",\n          1,\n          \"id\"\n        ],\n        \"value\": \"4TUswUJXBX1UzbauA8D6N8\",\n        \"details\": \"The property \\\"id\\\" is not allowed here.\"\n      },\n      {\n        \"name\": \"unknown\",\n        \"path\": [\n          \"fields\",\n          \"contributingCohorts\",\n          \"en-US\",\n          1,\n          \"name\"\n        ],\n        \"value\": \"BADGE-PD\",\n        \"details\": \"The property \\\"name\\\" is not allowed here.\"\n      }\n    ]\n  },\n  \"request\": {\n    \"url\": \"/spaces/6ekgyp1432o9/environments/gp2-3733/entries/7c8BeaLNSSFZhFlpBuiHxN\",\n    \"headers\": {\n      \"Accept\": \"application/json, text/plain, */*\",\n      \"Content-Type\": \"application/json-patch+json\",\n      \"X-Contentful-User-Agent\": \"sdk contentful-management.js/0.0.0-determined-by-semantic-release; platform node.js/v18.17.1; os Linux/v18.17.1;\",\n      \"Authorization\": \"Bearer ..._2BoU\",\n      \"X-Contentful-Version\": \"14\",\n      \"User-Agent\": \"axios/1.4.0\",\n      \"Content-Length\": \"1845\",\n      \"Accept-Encoding\": \"gzip, compress, deflate, br\"\n    },\n    \"method\": \"patch\",\n    \"payloadData\": \"[{\\\"op\\\":\\\"replace\\\",\\\"path\\\":\\\"/fields/authors\\\",\\\"value\\\":{\\\"en-US\\\":[{\\\"sys\\\":{\\\"type\\\":\\\"Link\\\",\\\"linkType\\\":\\\"Entry\\\",\\\"id\\\":\\\"2oRu2iqt1Pk5lFt6bOfdij\\\"}},{\\\"sys\\\":{\\\"type\\\":\\\"Link\\\",\\\"linkType\\\":\\\"Entry\\\",\\\"id\\\":\\\"2hApg3gq0323W9q7eXrPb6\\\"}}]}},{\\\"op\\\":\\\"replace\\\",\\\"path\\\":\\\"/fields/addedDate\\\",\\\"value\\\":{\\\"en-US\\\":\\\"2023-09-27T10:45:44.474Z\\\"}},{\\\"op\\\":\\\"replace\\\",\\\"path\\\":\\\"/fields/documentType\\\",\\\"value\\\":{\\\"en-US\\\":\\\"Article\\\"}},{\\\"op\\\":\\\"replace\\\",\\\"path\\\":\\\"/fields/link\\\",\\\"value\\\":{\\\"en-US\\\":\\\"http://www.fuehwfiuhewiuhfuiwe.com.br/iudhqwiudhuwqihdhiuq\\\"}},{\\\"op\\\":\\\"add\\\",\\\"path\\\":\\\"/fields/publishDate\\\",\\\"value\\\":{}},{\\\"op\\\":\\\"replace\\\",\\\"path\\\":\\\"/fields/subtype\\\",\\\"value\\\":{\\\"en-US\\\":\\\"Preprints\\\"}},{\\\"op\\\":\\\"replace\\\",\\\"path\\\":\\\"/fields/title\\\",\\\"value\\\":{\\\"en-US\\\":\\\"vdsvsdvwev\\\"}},{\\\"op\\\":\\\"replace\\\",\\\"path\\\":\\\"/fields/type\\\",\\\"value\\\":{\\\"en-US\\\":\\\"Research\\\"}},{\\\"op\\\":\\\"replace\\\",\\\"path\\\":\\\"/fields/updatedBy\\\",\\\"value\\\":{\\\"en-US\\\":{\\\"sys\\\":{\\\"type\\\":\\\"Link\\\",\\\"linkType\\\":\\\"Entry\\\",\\\"id\\\":\\\"6GHadgrQ1fI217B0pUjyFD\\\"}}}},{\\\"op\\\":\\\"replace\\\",\\\"path\\\":\\\"/fields/description\\\",\\\"value\\\":{\\\"en-US\\\":\\\"ewfewfwefewf\\\"}},{\\\"op\\\":\\\"replace\\\",\\\"path\\\":\\\"/fields/gp2Supported\\\",\\\"value\\\":{\\\"en-US\\\":\\\"Don't Know\\\"}},{\\\"op\\\":\\\"replace\\\",\\\"path\\\":\\\"/fields/sharingStatus\\\",\\\"value\\\":{\\\"en-US\\\":\\\"GP2 Only\\\"}},{\\\"op\\\":\\\"add\\\",\\\"path\\\":\\\"/fields/tags\\\",\\\"value\\\":{}},{\\\"op\\\":\\\"add\\\",\\\"path\\\":\\\"/fields/doi\\\",\\\"value\\\":{}},{\\\"op\\\":\\\"add\\\",\\\"path\\\":\\\"/fields/rrid\\\",\\\"value\\\":{}},{\\\"op\\\":\\\"add\\\",\\\"path\\\":\\\"/fields/accessionNumber\\\",\\\"value\\\":{}},{\\\"op\\\":\\\"add\\\",\\\"path\\\":\\\"/fields/contributingCohorts\\\",\\\"value\\\":{\\\"en-US\\\":[{\\\"id\\\":\\\"5kebYlHRiikmZuANOpMaHm\\\",\\\"name\\\":\\\"AGPDS\\\"},{\\\"id\\\":\\\"4TUswUJXBX1UzbauA8D6N8\\\",\\\"name\\\":\\\"BADGE-PD\\\"}]}},{\\\"op\\\":\\\"replace\\\",\\\"path\\\":\\\"/fields/relatedEntities\\\",\\\"value\\\":{\\\"en-US\\\":[{\\\"sys\\\":{\\\"type\\\":\\\"Link\\\",\\\"linkType\\\":\\\"Entry\\\",\\\"id\\\":\\\"7zAIlUdb8KX66ZkwOky8xM\\\"}},{\\\"sys\\\":{\\\"type\\\":\\\"Link\\\",\\\"linkType\\\":\\\"Entry\\\",\\\"id\\\":\\\"7dQkSMSEGMTP4FFjl2d7z\\\"}},{\\\"sys\\\":{\\\"type\\\":\\\"Link\\\",\\\"linkType\\\":\\\"Entry\\\",\\\"id\\\":\\\"9t9UyhWo9jQdVo0QcG9xC\\\"}},{\\\"sys\\\":{\\\"type\\\":\\\"Link\\\",\\\"linkType\\\":\\\"Entry\\\",\\\"id\\\":\\\"2J4aQzK06FTDoMjo9KKbaY\\\"}}]}}]\"\n  },\n  \"requestId\": \"e2a09633-9106-41ef-ba55-886c4b44cb8a\"\n}",
    "statusCode": 500
}
```